server/config.js

```diff
@@ -52,6 +52,11 @@ module.exports = {
   test: isTest,
   production: isProduction,
   backendUrl,
+  buildInfo: {
```

Comment:
I'm wondering if we can be more explicit with these default values?

Reply:
We need something fixed and immediately identifiable as not prod but otherwise open to suggestions 😄

Reply:
Other than `unset`, none immediately off the top of my head. But I agree we should focus on the fix at hand